README.md

```diff
@@ -1 +1,86 @@
-Put all your good documentation here. 
+Welcome to Risknet! This is a downloadable Pip package where you can access and run an XGBoost pipeline.
+
+# Layout
+- src/risknet
+   - config: holds `conf.yaml`
+      - `conf.yaml`: helps with setup
+      - `handlers.py`
+   - data: empty folder where user can store FM dataset
+   - jobs
+      - `__init__.py`
+      - `cloud_etl.py`: helps save files to cloud
+   - proc: contains preprocessing steps like feature encoding, label prep, and train-test splits
+      - `__init__.py`
+      - `encoder.py`: feature engineering/encoding categoricals
+      - `label_prep.py`: defines default, progress on loan
+      - `reducer.py`: reduces features based on importance, also train/test/val splits
+   - run: contains files for running pipeline
+      - `__init__.py`
+      - `main.py`: currently empty. WIP define pipeline as a function and call here
+      - `model.py`: defines the model class + functions
+      - `pipeline.py`: calls functions to execute the pipeline
+   - sys: contains files to set up system environment and logging
+      - `__init__.py`
+      - `log.py`: sets up logger
+      - `managers.py`: sets up the dask manager
+      - `system.py`: defines creating and removing files via the `sys` package
+   -`init.py`
+   -`main.py`: logs start, stop time for running the program (including downloading packages from setup.cfg)
+   - tests: store tests here
+      -`__init__.py`
+      -`test_stub.py`: currently only asserts True == True. No tests added yet.
+
+# Steps to Update Version
+To update the version:
+- `rm -rf dist build` to remove build folder
+- manually remove "egg-info" folder. This will change `src` to `src/risknet`.
+- update setup.cfg's version number depending on if major, minor, or bug change
+- rerun `python3 -m build` (you should get a new dist folder + egg folder in \src)
+   - THIS SHOULD CREATE A NEW binary file where version is UPDATED
+- rerun `python3 -m twine upload --repository testpypi dist/*`
+   - Username: __token__
+   - Password: [testpypi password starting with pypi]
+   - If you did NOT update the version # before running `build` then you will get an error
+
+
+# History:
+0.0.1: Ran into problems with installing pytest-cov
+
+0.0.2: Got error:
+`ERROR: Could not find a version that satisfies the requirement dask[complete] (from risknet) (from versions: none) ERROR: No matching distribution found for dask[complete]`
+
+0.0.3: Tried moving dask into [options.extras_require] not install_requires. Got error for flake8
+
+0.0.4: Commented out more packages.
+Got error: `error: package directory 'lib3/yaml' does not exist` and `metadata-generation-failed`.
+
+0.0.5: Moved types-YAML into options, still get `'lib3/yaml does not exist'` error :/
```

Comment:
Probably minor releases like these we don't have to have a public facing changeling for...

Reply:
I updated version numbers regularly (VERY regularly 😭) because to compile/build with testpypi, you need to change the version history every time code changes (otherwise you get an error).

Not sure how to make minor releases WITHOUT public-facing changes -- maybe we can discuss in next meeting?